_posts/2021-12-27-trino-2021-a-year-of-growth.md

```diff
@@ -0,0 +1,346 @@
+of `JSON` arrays.  
+
+## Performance Improvements
+
+* Improved performance of querying Parquet data for files containing column indexes.
+* Improved performance when reading dictionary-encoded Parquet files.
+* Improved performance of queries using [`rank()`](https://trino.io/docs/current/functions/window.html#rank) `WINDOW` function.
+* Improved performance of [`sum()`](https://trino.io/docs/current/functions/aggregate.html#sum)
+and [`avg()`](https://trino.io/docs/current/functions/aggregate.html#avg) for 
+decimal types.
+* Improved performance of `GROUP BY` with single grouping column.
+* Improved performance of decimal aggregations.
+* Improved performance when evaluating the WHERE and SELECT clause.
+* Improved performance when computing the product of decimal values with 
+precision larger than 19.
+* Improved performance of queries that process row or array data.
+* Reduced memory usage for processing `JOIN` clauses.
+* Improved performance of queries that contain a `DISTINCT` clause. 
+
+## Security
+
+* Added support for automatic configuration of TLS for 
+[Secure internal communication](https://trino.io/docs/current/security/internal-communication.html).
+This removes the need to provision per-worker TLS certificates.
+* Added support to use internal names for discovery client when automatic TLS is
+enabled for internal communications. This allows discovery.uri to be configured 
+using a normal DNS name like `https://coordinator.trino` and still use automatic
+TLS certificates.
+* Added support for OAuth2/OIDC opaque access tokens.
+* Added support for configuring multiple password authentication plugins.
+* Added property to hide inaccessible columns from `SELECT *` statement when the 
+`hide-inaccessible-columns` configuration property is set to true. 
+* Allowed configuring HTTP proxy for OAuth2 authentication.
+* Added `IN catalog` clause to `CREATE ROLE`, `DROP ROLE`, `GRANT ROLE`, 
+`REVOKE ROLE`, and `SET ROLE` to specify the target catalog of the statement 
+instead of using the current session catalog.
+* Fix handling of SNI for multiple TLS certificates.
+
+## Data Sources
+
+### BigQuery Connector
+
+* Added support for case insensitive name matching for BigQuery views.
+* Changed type mapping of BigQuery datetime from `timestamp(3)` to `timestamp(6)`
+in Trino.
+* Changed type mapping of BigQuery timestamp from `timestamp(3)` with time zone 
+to `timestamp(6)` with time zone in Trino.
+* Fixed failures for queries accessing `information_schema.columns` when 
+`case-insensitive-name-matching` is disabled.
+
+### Cassandra connector
+
+* Mapped Cassandra `uuid` type to Trino `uuid`.
+* Added support for Cassandra `tuple` type.
+* Changed minimum number of speculative executions from 2 to 1. 
+
+### Clickhouse connector (new)
+
+* Added [ClickHouse connector](https://trino.io/docs/current/connector/clickhouse.html).
+* Added `write.batch-size` connector configuration property to control JDBC batch 
```

Comment:
Dont repeat the docs and talk about property and session property names .. just talk about the feature ... here and all following items